### PR TITLE
ci: change to nx release

### PR DESCRIPTION
**Pull request by ju-Skinner**
# Description
Change over from bare metal NPM publish to NX Release.

### CODE REVIEW

File: .github/workflows/actions/setup/action.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    - name: Get NPM cache directory
+      id: npm-cache-dir
+      shell: bash
+      run: echo "dir=$(npm config get cache)" >> $GITHUB_ENV
+
+    - name: Configure user
+      run: |
+        git config user.name "DSS Automation Bot"
+        git config user.email "dev+github-bot@kajabi.com"
```
Comment (monicawheeler) [inline .github/workflows/actions/setup/action.yml:33]: Do we need a dedicated `dss+github-bot@kajabi.com` or is this a catch-all email that is fine to use for any workflow?

File: .github/workflows/schedule-release.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+      ref:
+        type: string
+        description: The branch name, tag, or SHA to be checked out. This can also be left blank.
+        default: ''
+
+      tag:
+        type: choice
+        required: true
+        description: Which npm tag should this be published to
```
Comment (monicawheeler) [inline .github/workflows/schedule-release.yml]:
```suggestion
        description: Which npm tag should this be published to?
```

File: .github/workflows/schedule-release.yml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+        uses: ./.github/workflows/actions/setup
+
+      - name: Restore core built Cache
+        uses: ./.github/workflows/actions/download-archive
+        with:
+          name: pine-core
+          path: libs/core
+          filename: CoreBuild.zip
+
+      - name: Restore react built Cache
```
Comment (monicawheeler) [inline .github/workflows/schedule-release.yml]:
```Suggestion
      - name: Restore React built Cache
```

**Review by monicawheeler — APPROVED** [review]
🙌🏻  Just a few non-blocking comments!

File: .github/workflows/actions/setup/action.yml (hunk 1 above)
Comment (ju-Skinner) [reply .github/workflows/actions/setup/action.yml:33]: This is not a real email that we use. It's for bot purposes.

**Review by QuintonJason — APPROVED** [review]

**Comment by QuintonJason** [comment]
Good to go after rebase

**Pull request merged**